content/python/concepts/strings/terms/endswith/endswith.md

```diff
@@ -0,0 +1,72 @@
+---
+Title: '.endswith()'
+Description: 'Checks whether or not a string ends with the given value.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Functions'
+  - 'Methods'
+  - 'Strings'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+  - 'paths/data-science'
+---
+
+TThe `.endswith()` method returns `True` if a particular value is attached to the end of a given string. Otherwise, it returns `False`.
+
+## Syntax
+
+Given a  `string`, the `.endswith()` method can be used in the following way:
+
+The `value` is required argument that is checked for whether it exists at the beginning of the `string`.  It is also case-sensitive.
+
+Conversely, the `value` can be tested whether it is the end of a particular substring of `string` by using the optional `start` and `end` index arguments.
+
+## Examples
+
+The following checks for the value `g` at the end of the `example` string:
+
```

Comment:
I think it would. be better to consolidate these snippets into two: 

* One code snippet for the Python. \```py
* One output snippet for the printed boolean values. \```shell
* After the snippets should be 2-3 sentences explaining how each `check_` gets its value.

```py
example_str = "This is a string"

check_A = example_str.endswith("g")
check_B = example_str.endswith("s")
check_C = example_str.endswith("st", 5, 12)

print("A: ", check_A)
print("B: ", check_B)
print("C: ", check_C)
```

And the output would be:

```shell
A: True
B: False
C: True
```

Reply:
Just looked at it and it's looking better! 👍 I left one more suggestion for the snippet below.